src/component/Header/index.jsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Nav from 'react-bootstrap/Nav';
 import Navbar from 'react-bootstrap/Navbar';
+import Modal from 'react-bootstrap/Modal';
+import Button from 'react-bootstrap/Button';
 import Image from 'react-bootstrap/Image';
+import SettingForm from '../SettingForm';
 import styles from './Header.module.css';
 import pomoLogo from '../../Logo/PomoLogo.png';
 import settingLogo from '../../Logo/SettingLogo.png';
 import loginLogo from '../../Logo/LoginLogo.png';
 
+function SettingModal() {
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Button variant="outline-light" className="mb-3" onClick={handleShow}>
+        <i>setting</i>
+      </Button>
+      <Modal show={show} onHide={handleClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Setting</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <SettingForm />
+        </Modal.Body>
+      </Modal>
+    </>
+  );
+}
```

Comment:
Each component should be placed in a separate file. Can you move this to `src/component/SettingsModal`?